Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -54,9 +54,16 @@ def PyInit(self):
                              direction=Direction.Output),
                              ("Name the workspace that will contain the " +
                               "calibration coefficients"))
+        self.declareProperty("Temperature",
+                             defaultValue=Property.EMPTY_DBL,
+                             validator=FloatBoundedValidator(lower=0.0),
+                             direction=Direction.Input,
+                             doc=("Temperature during the experiment."))
         return
 
     def validateInputs(self):
+        """ Validate the inputs
+        """
         issues = dict()
         inws = self.getProperty("VanadiumWorkspace").value
         run = inws.getRun()
```

Comment:
Interestingly, I'm having trouble recreating it now.  The first time I didn't have all the workspaces in the right format for a few times before I got it right.

I'd be  tempted just to put in 
``` python 
if inws is not None:
```